samples/dnn/segmentation.py

```diff
@@ -102,7 +102,7 @@ def showLegend(classes):
     inpWidth = args.width if args.width else frameWidth
     inpHeight = args.height if args.height else frameHeight
     blob = cv.dnn.blobFromImage(frame, args.scale, (inpWidth, inpHeight), args.mean, args.rgb, crop=False)
-
+    
```

Comment:
Please remove trailing witespaces

Reply:
Hello sir, I had removed the whitespace, thank you!